src/Symfony/Component/CssSelector/Converter.php

```diff
@@ -19,11 +19,48 @@
 use Symfony\Component\CssSelector\XPath\Translator;
 
 /**
- * @author Christophe Coevoet <stof@notk.org>
+ * Converter is the main entry point of the component and can convert CSS
+ * selectors to XPath expressions.
+ *
+ * This component is a port of the Python cssselect library,
+ * which is copyright Ian Bicking, @see https://github.com/SimonSapin/cssselect.
+ *
+ * Copyright (c) 2007-2012 Ian Bicking and contributors. See AUTHORS
```

Comment:
@fabpot is the phpdoc the right place to put the license of the ported library ? This class is not even ported from the Python library (the classes porting the Pythin library are all the internal classes)